src/Model/VggWork.cpp

```diff
@@ -0,0 +1,99 @@
+#include "VggWork.hpp"
+
+#include "Utils/Utils.hpp"
+
+#include <miniz-cpp/zip_file.hpp>
```

Comment:
`lib/miniz-cpp` will be deprecated soon and should not used in new code anymore. It exists because the old sketch parser depends on it. There will be a code clean-up soon.

New zip lib in `lib/vgg_contrib/zip` should be used instead.